functions/calculate-date/1.0/index.js

```diff
@@ -0,0 +1,25 @@
+import moment from 'moment';
+
+const calculateDate = async ({ method, date, years, months, weeks, days }) => {
+  if (method === 'add') {
+    return {
+      as: moment(date, 'YYYY-MM-DD')
```

Comment:
Almost there :) declare `as` make two oneliners arrow functions outside of this function that do this and have a ternary in this function that can call one of the two functions depending on the if statement above